From 0f72787b922c53c33d497b17300a959b911e621f Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Tue, 14 Feb 2017 20:12:05 +0100
Subject: [PATCH] io.cc: add stdio.h include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes buildroot compile failure with uclibc [1]:

  In file included from io.cc:28:0:
  block.h:219:22: error: ‘FILE’ has not been declared
     int write_mapfile( FILE * f = 0, const bool timestamp = false,

[1] http://autobuild.buildroot.net/results/4ac0754f1cc5ea934d6437e89d1f4906fb3fd0a8

Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 io.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/io.cc b/io.cc
index 36b8341..2d6a76e 100644
--- a/io.cc
+++ b/io.cc
@@ -23,6 +23,7 @@
 #include <string>
 #include <vector>
 #include <stdint.h>
+#include <stdio.h>
 #include <unistd.h>
 
 #include "block.h"
-- 
2.11.0

